document/js-api/index.bs

```diff
@@ -1120,26 +1144,155 @@ The algorithm <dfn>ToWebAssemblyValue</dfn>(|v|, |type|) coerces a JavaScript va
 1. If |type| is [=f64=],
     1. Let |f64| be ? [=ToNumber=](|v|).
     1. Return [=f64.const=] |f64|.
-1. If |type| is [=funcref=],
+1. If |type| is of the form [=ref=] |null| |heaptype| and is a subtype of [=ref=] |null| [=heap-type/func=],
```

Comment:
It looks like you may be able to avoid some duplication by first checking the null case, and only then distinguish between func/extern/any cases.

Reply:
Thanks, fixed in the latest commit.